pkgs/applications/video/mpv/default.nix

```diff
@@ -13,25 +13,25 @@
     wayland      ? null,
     libxkbcommon ? null
 
-, rubberbandSupport  ? !stdenv.isDarwin, rubberband ? null
+, rubberbandSupport  ? true,  rubberband    ? null
 , xineramaSupport    ? true,  libXinerama   ? null
 , xvSupport          ? true,  libXv         ? null
 , sdl2Support        ? true,  SDL2          ? null
-, alsaSupport        ? !stdenv.isDarwin,  alsaLib       ? null
```

Comment:
Why is this removed?

Reply:
I moved these to the package call (in all-packages.nix) since there was a `waylandSupport = stdenv.isLinux;` there already.